packages/manager/apps/account-disable-2fa/src/utils/iframe.ts

```diff
@@ -0,0 +1,53 @@
+export type IframeLoaderPayload = {
+  src: string;
+  id?: string;
+  title?: string;
+  style?: string;
+  width?: string;
+};
+
+const getNewIframeElement = (
+  payload: IframeLoaderPayload,
+): HTMLIFrameElement => {
+  if (!payload || !payload.src) {
+    throw new Error('[IframeLoader][getNewIframeElement()] missing payload.');
+  }
+
+  const iframe = document.createElement('iframe');
+  iframe.src = payload.src;
+
+  if (payload.title) {
+    iframe.title = payload.title;
+  }
+  if (payload.width) {
+    iframe.width = payload.width;
+  }
+  if (payload.id) {
+    iframe.id = payload.id;
+  }
+  if (payload.style) {
+    iframe.style.cssText = payload.style;
+  }
+
+  return iframe;
+};
+
+export const loadIframe = (payload: IframeLoaderPayload): Promise<unknown> => {
+  if (!payload || !payload.src) {
+    throw new Error('[IframeLoader][load()] missing payload.');
+  }
```

Comment:
**nitpick:** Since the check is also added on `getNewIframeElement`, we can remove it.